cmd/kubeadm/app/cmd/phases/reset/cleanupnode.go

```diff
@@ -135,7 +138,7 @@ func runCleanupNode(c workflow.RunData) error {
 func absoluteKubeletRunDirectory() (string, error) {
```

Comment:
Is this function still needed?

`absoluteKubeletRunDirectory()` == `filepath.EvalSymlinks(kubeadmconstants.KubeletRunDirectory)`.

Reply:
updated.